File: 1-Iniciante/1035-Teste_de_selecao_1-2/1035.py
nuns=input().split(' ')
a= int(nuns[0])
b= int(nuns[1])
c= int(nuns[2])
d= int(nuns[3])
if b > c and d > a and c+d > a+b and c>0 and d>0 and a%2==0:
    print("Valores aceitos")
else:
    print("Valores nao aceitos")

File: 1-Iniciante/1035-Teste_de_selecao_1-2/1035.cs
using System;

public class HelloWorld
{
    public static void Main(string[] args)
    {
        string[] x = Console.ReadLine().Split(' ');
        
        int a = int.Parse(x[0]);
        int b = int.Parse(x[1]);
        int c = int.Parse(x[2]);
        int d = int.Parse(x[3]);
        
        if (b > c & d > a & c+d > a+b & c>0 & d>0 & a%2==0){
            Console.WriteLine("Valores aceitos");
        }
        else{
            Console.WriteLine("Valores nao aceitos");
        }
    }
}